### PR TITLE
seald identity setup

### PR DESCRIPTION
**Pull request by bluquar**
https://user-images.githubusercontent.com/4309735/144299234-e90eda64-47fa-4b21-87ee-97ac15d04896.mov

### CODE REVIEW

**Comment by bluquar** [comment]
Next steps:
1. When loading the current encryption state, show a full-screen "Loading encryption data" from Navigation.tsx (right after the empty state when we're loading account data.
2. If username is null, skip that loading state
3. Also show "checking login status" loading screen at the top of Navigation
4. That way we can assume we're logged in and encryption data is loaded whenever we're under the Root navigation

**Review by lowellbander — APPROVED** [review]
will you walk me through the encryption model verbally, perhaps on a whiteboard? I'd really love to have a solid understanding of it!

**Comment by bluquar** [comment]
Yep! I could use a better understanding of it too

File: src/ca/screens/PersonDetailsScreen.tsx [hunk 1]
```diff
@@ -62,10 +63,13 @@ export default function PersonDetailsScreen({
     );
   }
 
+  const identity = PersistentClientStorage.get('identity');
```
[the remark this answers is not in the document]
Comment (bluquar) [reply src/ca/screens/PersonDetailsScreen.tsx:66]: yes, this actually doesn't need to be here though. I'll remove it in the next PR

**Pull request merged**